### DISCUSSION TITLE
Adding the Lib files inside a Script ver. of BTN

### DISCUSSION THREAD

**Discussion by AlecMcCutcheon** [Windos/BurntToast#205] [Q&A]
Hello, I recently forked the repo https://github.com/AlecMcCutcheon/Portable-BurntToast, to try and convert the module into a script with all the functions embedded inside, so that it could be pulled in and used temporarily without having to be installed. that part is done but I realized that I also need to either embed the libs in the libs folder in the script or pull them from the internet, both would end up being a string in the script, my question is how can I "import" or use the libs once it's a string in the script.

**Comment by Windos** [comment]
Hi there @AlecMcCutcheon, I'm not sure if that'd be possible without eventually having files on disk.

You _could_ base64 encode the dlls, include that string in the script, then decode and write them back to disk as files but I think that defeats the purpose of what you're going for?

**Reply by AlecMcCutcheon** [reply]
Well what are the commands for adding the libs, I've been able to add dll modules direct from the internet before using this method: 

Import-Module ([System.Reflection.Assembly]::Load((Invoke-WebRequest -UseBasicParsing -Uri "Your DLL Module File URL").content)) 

but they aren't modules so it's probably a bit different, but I hadn't seen anyone load dll code into the import-module function before I found that trick, so if I knew all the ways the libs can be "imported" so to speak, then maybe I can find a trick for that too, I'm just not familiar with loading libs to know the commands used.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
